### ISSUE TITLE
IPv6

### ISSUE THREAD

**Issue by nicraMarcin** [hackerspace-silesia/Smogomierz#105]
Witam

czy przewidywane jest wsparcie dla IPv6?

**Comment by bfaliszek** [comment]
Hej, w którym przypadku IPv6 nie działa?

**Comment by nicraMarcin** [comment]
Uruchomiłem smogomierz. W konsoli widzę, że tylko IPv4 zostaje przyznane. Nigdzie nie widzę możliwości skonfigurowania IPv6.

**Comment by bfaliszek** [comment]
A masz skonfigurowane IPv6 na routerze i innym urządzenim adresy są przydzielane?

**Comment by nicraMarcin** [comment]
Tak, na innych urządzeniach w sieci mi IPv6 działa bez problemu.
